editoast/src/views/test_app.rs

```diff
@@ -9,12 +9,17 @@ use axum_tracing_opentelemetry::middleware::OtelAxumLayer;
 use dashmap::DashMap;
 use editoast_models::DbConnectionPoolV2;
 use editoast_osrdyne_client::OsrdyneClient;
+use futures::future::BoxFuture;
+use opentelemetry_sdk::export::trace::ExportResult;
+use opentelemetry_sdk::export::trace::SpanData;
+use opentelemetry_sdk::export::trace::SpanExporter;
 use serde::de::DeserializeOwned;
 use tower_http::trace::TraceLayer;
 use url::Url;
 
 use crate::{
     core::{mocking::MockingClient, CoreClient},
+    create_tracing_subscriber,
```

Comment:
This is the import that breaks the dependency order. `views` cannot `use` something from the `crate` or `client` level. I'd suggest moving `create_tracing_subscriber` to `editoast_common`.

Reply:
Done.